lib/db.js

```diff
@@ -4,10 +4,27 @@ export const foodCategories = [
     name: "Vollkorngetreide",
     slug: "vollkorngetreide",
     recommendedConsumption: 232,
-    recommendedExample:
-      "2 Tassen Vollkornreis und 1,5 Scheiben Vollkornroggenbrot",
-    maxRange: 464,
-    maxRangeInputField: 600,
+    recommendedExample: "1 Tasse Vollkornreis und 1 Scheibe Vollkornroggenbrot",
+    maxRa600nge: 464,
```

Comment:
shouldn't this be `maxRange` instead of `maxRa600nge`?

Reply:
yes!! Thanks alot!!